linux/debian/jamulus-headless.postinst

```diff
@@ -6,4 +6,7 @@ set -e
 
 adduser --system --quiet --home /nonexistent --no-create-home jamulus
 
+sudo mkdir /etc/jamulus
+sudo touch /etc/jamulus/welcome.html
```

Comment:
This will require the user to edit the file as `root`.  The file needs to be readable by the user `jamulus`, and - depending on how the system is already set up - this may not be the case.

Reply:
How about:

**chmod -c 666 /path/to/file**

Everyone can read and write, nobody can execute.

Reply:
I wouldn't want files installed on a system that any user could edit without control, that were then displayed publicly...

Ideally:
* there would be a `jamulus` _group_ as well as user
* the file would be placed in a location owned by the `jamulus` user and owned by `jamulus:jamulus`
* permissions could then be 0660 - so `jamulus` user and any user in the `jamulus` group could edit it - but no one else

This is getting outside the scope of this PR, though, so I think just leaving (with `-p`) will do for now - so only `root` can edit.  It probably _is_ worth making use it's `a+r` permissions (i.e. _anyone_ can read).

Reply:
a+r appears to work

Reply:
Check jamulus.install file and deploy_deb.sh. I think you just need to move the file to the correct location during deb creation. This ensures the file doesn't get overwritten by updates, I suppose

Reply:
touch can't overwrite, and this file has no contents by design. what would overwrite it?

Reply:
I don't want my custom welcome file to be discarded when I upgrade.

Reply:
Exactly. I think that we might want to add an empty file to the deb directly instead of creating it during install time. But I'm unsure about the actual way config files are handled. 

For example I'd like that a apt purge deletes it while an apt remove doesn't. 

I found https://wiki.debian.org/ConfigPackages but it doesn't help me. 

@mirabilos how should config files be handled correctly in deb files?

Reply:
The usual Debian way: ship it as conffile by placing it in `/etc/jamulus/` in the package contents, then it will be placed there on first install by `dpkg` and later updated with newer versions iff the user did not change it; if the user changed it, they’ll get a chance to merge, keep their changes or throw their changes away. And the file will have proper default `root:root 0644` permissions.

Reply:
@ann0see, touch won't overwrite anything. This file would behave more appropriately with apt purge vs. apt remove, and I guess that's good citizenship. Someday deploying default welcome file metadata could be a thing.

Reply:
@mirabilos thanks for answering.